### PR TITLE
Playtest fixes3

### LINKED ISSUE #8347: Character models spin unintentionally
### Environment
- Report date: February 4th, 2022 
- OS Version: Multiple

### Detailed description of bug

Character models spin around at random. According to @Bod9001, this is down to the "Upright sprite scripts" lacking supports for being rotatable. See here: https://discord.com/channels/273774715741667329/668605946691452949/939304431835570226

### PR DESCRIPTION
**Pull request by Bod9001**
fixes rotatable getting updates from Server for player direction #8347 
CL: [Fix] fixed fire locks not being openable with crowbars

### CODE REVIEW

**Comment by corp-0** [comment]
does it close any issue?

**Comment by Bod9001** [comment]
can't find issue for fire locks not being crowerbarble

File: UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs [hunk 1]
```diff
@@ -36,6 +36,10 @@ public enum RotationMethod
 	[SyncVar(hook = nameof(SyncServerLockAndDirection))]
 	private LockAndDirection SynchroniseCurrentLockAndDirection;
 
+	[SerializeField]
+	[Tooltip("If the direction is only handled by client")]
+	private bool updateFromServer = true;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs:41]: I don't understand this. Name of the variable suggests the server handles it, the tooltip says the contrary

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs:41]: I'll update the tooltip

**Review by corp-0 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**